libbeat/scripts/Makefile

```diff
@@ -132,14 +133,14 @@ check: check-headers python-env prepare-tests ## @build Checks project and sourc
 .PHONY: $(.OVER)check-headers
 $(.OVER)check-headers:
 ifndef CHECK_HEADERS_DISABLED
-	@go get -u github.com/elastic/go-licenser
+	@go get $(GO_LICENSER_REPO)
 	@go-licenser -d -license ${LICENSE}
 endif
```

Comment:
I know it's a little out of scope, but constant dripping wears away the stone. Can we move check-headers to mage? Mage should also be able to get dependencies.